gnucash/import-export/csv-exp/csv-transaction-export-line.cpp

```diff
@@ -249,9 +249,11 @@ CsvTransactionExportLine::get_commodity(Transaction *trans)
 
 // Amount with Symbol or not
 std::string
-CsvTransactionExportLine::get_amount(Split *split, bool t_void, bool symbol)
+CsvTransactionExportLine::get_amount(Split *split, bool t_void, bool symbol, bool positive)
```

Comment:
Multiple boolean function args is poor design, it's too easy to screw up the order.